cmd/mc-router/main.go

```diff
@@ -124,6 +126,14 @@ func main() {
 
 	metricsBuilder := NewMetricsBuilder(config.MetricsBackend, &config.MetricsBackendConfig)
 
+	downScalerEnabled := config.AutoScaleDown && (config.InKubeCluster || config.KubeConfig != "")
+	downScalerDelay, err := time.ParseDuration(config.AutoScaleDownAfter)
+	if err != nil {
+		logrus.WithError(err).Error("Unable to parse down scale after duration")
```

Comment:
```suggestion
		logrus.WithError(err).Error("Unable to parse down scale after duration")
		downScalerDelay = 10 * time.Minute
```
Sorry if I’m misunderstanding, but shouldn’t downScalerDelay be set to the default value if the config is in an incorrect format?

Reply:
Actually, I was planning to make this `Fatal` so the program exists (I want the user to know they set an incorrect value rather than making it more difficult to figure out by only emitting the log).

Reply:
Yeah, I think a fatal would be a choice here.